### ISSUE TITLE
[Marketing]  YouTube

### ISSUE THREAD

**Issue by MikeTheWatchGuy** [fsmosca/Python-Easy-Chess-GUI#31]
There are some pretty darned popular YouTube channels devoted to chess and some with chess and AI.  It seems like if you could get just one of those guys / gals (one chick in particular I found quite, hum, entertaining). 

I wonder if any of them would jump at having custom software just for them.  You can easily add their logo or YouTube name to the Window somewhere to make it extra exclusive just for them.  Heck, maybe the fact they have input into something like that may be enough to get their attention.

I dunno, just throwing random stuff out there.  May or not not be helpful in any way.  As usual, the price of this advice should tell you a little about its potential value ;-)

**Comment by MikeTheWatchGuy** [comment]
It also seems to me that you know a lot about AI and chess.  Do YOU happen to have a YouTube channel already or maybe should have one?  If so, heck, you've got your own software to help.

If you're as bright as you seem to me to be, and you are able to glue together different AI engines, then you could easily "own" the space of software that interfaces with and pits head to head the chess engines of the day.

Or maybe you're able to do something with Google's Alpha stuff?

**Comment by MikeTheWatchGuy** [comment]
In fact, AI is one of the big drivers behind PySimpleGUI.   Despite it not yet been integrated very well, I have integrated with OpenCV which opened up all sorts of stuff.  It made making a [YOLO object identification program](https://github.com/PySimpleGUI/PySimpleGUI/tree/master/YoloObjectDetection) downright **_easy_** to do.  It honestly was easy.

I recently did something similar with a bird flocking behavior modeling software named boid.  It was running on a GUI framework named P5.  It was easy to adapt his software to interface with PySimpleGUI.  It used the PySimpleGUI basic drawing primitives of drawing a small circle on a Graph Element.

Seeing how iconic Chess is and the following it has as well + all the recent and wild AI work that's happened with Chess it would seem there would be room for something to help evaluate these engines or pit them against each other or run really long numbers of games.   Or ......
